### ISSUE TITLE
win11 WSL cp的时候提示目录不存在

### ISSUE THREAD

**Issue by wzqvip** [lgl88911/riscv_tldr#1]
手动下载了仓库
apt安装tldr
然后复制出错

看了下发现home/用户/ 下并没有.tldr，常见的地方也没查到，想问一问这个应该存储在什么地方

```
taco@Taco-Stick:~/Files/riscv_tldr$ sudo apt-get install tldr
[sudo] password for taco:
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libfwupdplugin1 libxmlb1
Use 'sudo apt autoremove' to remove them.
The following NEW packages will be installed:
  tldr
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 584 kB of archives.
After this operation, 3013 kB of additional disk space will be used.
Get:1 http://mirrors.tuna.tsinghua.edu.cn/ubuntu focal/universe amd64 tldr amd64 0.4.0.1-1build2 [584 kB]
Fetched 584 kB in 0s (1640 kB/s)
Selecting previously unselected package tldr.
(Reading database ... 78839 files and directories currently installed.)
Preparing to unpack .../tldr_0.4.0.1-1build2_amd64.deb ...
Unpacking tldr (0.4.0.1-1build2) ...
Setting up tldr (0.4.0.1-1build2) ...
update-alternatives: using /usr/bin/tldr-hs to provide /usr/bin/tldr (tldr) in auto mode
Processing triggers for man-db (2.9.1-1) ...
taco@Taco-Stick:~/Files/riscv_tldr$ cd ..
taco@Taco-Stick:~/Files$ cp riscv_tldr/rv32i/* ~/.tldr/tldr/pages/common/
cp: target '/home/taco/.tldr/tldr/pages/common/' is not a directory
taco@Taco-Stick:~/Files$
taco@Taco-Stick:~/Files$ cp riscv_tldr/pseudo/* ~/.tldr/tldr/pages/common/
cp: target '/home/taco/.tldr/tldr/pages/common/' is not a directory

```

**Comment by arbaleast** [comment]
不清楚，tldr 该怎么配置，可以尝试用cheat
md 文档改下名也能用
> 手动下载了仓库 apt安装tldr 然后复制出错
> 
> 看了下发现home/用户/ 下并没有.tldr，常见的地方也没查到，想问一问这个应该存储在什么地方
> 
> ```
> taco@Taco-Stick:~/Files/riscv_tldr$ sudo apt-get install tldr
> [sudo] password for taco:
> Reading package lists... Done
> Building dependency tree
> Reading state information... Done
> The following packages were automatically installed and are no longer required:
>   libfwupdplugin1 libxmlb1
> Use 'sudo apt autoremove' to remove them.
> The following NEW packages will be installed:
>   tldr
> 0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
> Need to get 584 kB of archives.
> After this operation, 3013 kB of additional disk space will be used.
> Get:1 http://mirrors.tuna.tsinghua.edu.cn/ubuntu focal/universe amd64 tldr amd64 0.4.0.1-1build2 [584 kB]
> Fetched 584 kB in 0s (1640 kB/s)
> Selecting previously unselected package tldr.
> (Reading database ... 78839 files and directories currently installed.)
> Preparing to unpack .../tldr_0.4.0.1-1build2_amd64.deb ...
> Unpacking tldr (0.4.0.1-1build2) ...
> Setting up tldr (0.4.0.1-1build2) ...
> update-alternatives: using /usr/bin/tldr-hs to provide /usr/bin/tldr (tldr) in auto mode
> Processing triggers for man-db (2.9.1-1) ...
> taco@Taco-Stick:~/Files/riscv_tldr$ cd ..
> taco@Taco-Stick:~/Files$ cp riscv_tldr/rv32i/* ~/.tldr/tldr/pages/common/
> cp: target '/home/taco/.tldr/tldr/pages/common/' is not a directory
> taco@Taco-Stick:~/Files$
> taco@Taco-Stick:~/Files$ cp riscv_tldr/pseudo/* ~/.tldr/tldr/pages/common/
> cp: target '/home/taco/.tldr/tldr/pages/common/' is not a directory
> ```

**Comment by ccnuktd** [comment]
可以采用这个[链接](https://zhuanlan.zhihu.com/p/559057595)的方式安装tldr
安装好了以后就可以在home目录中找到.tldr目录，我们需要把原来的`rv32i`和`pseudo`两个文件放在`~/.tldr/
cache/pages/common/`下面，最后使用`tldr -u`更新cache即可
